### PR TITLE
[inventory] updates for bionic

### PR DESCRIPTION
**Pull request by adborden**
Updates and fixes to support Bionic.

### CODE REVIEW

**Comment by adborden** [comment]
Rebased to resolve merge conflict.

File: ansible/inventories/bionic/group_vars/all/vars.yml [hunk 1]
```diff
@@ -124,8 +124,8 @@ postfix_pem: "{{ default_tls_host_certificate }}"
 saml2_idp_entry: login.test.max.gov
 saml2_idp_metadata_url: "https://{{ saml2_idp_entry }}/idp/shibboleth"
 saml_idp_cert_fprint: "7A43DACB95D67FB6A804FD37C8A7DCA5B0384939"
-saml2_sp_public_key: "{{ default_tls_host_key }}"
-saml2_sp_private_pem: "{{ default_tls_host_certificate }}"
+saml2_sp_public_key: "{{ default_tls_host_certificate }}"
+saml2_sp_private_pem: "{{ default_tls_host_key }}"
```
Comment (pburkholder) [inline ansible/inventories/bionic/group_vars/all/vars.yml]: The naming here seems wrong with `_key` -> `_certificate` and `_pem` -> `_key`. Is this a mistake, or just really confusing naming?

File: ansible/inventories/production/hosts [hunk 2]
```diff
@@ -62,7 +62,7 @@ dashboardweb[1:2]p.prod-ocsit.bsp.gsa.gov
 [elasticsearch]
 
 [inventory-web:children]
-inventory-web-v1
+inventory-web-v1 datagov_in_service=false
```
Comment (pburkholder) [inline ansible/inventories/production/hosts:65]: Just a 👍 for this logic instead of the `!v2` in the `inventory.yml`

**Review by pburkholder — COMMENTED** [review]
Let's confirm the SAML2 _key, _cert, _pem, naming confusion

File: ansible/inventories/bionic/group_vars/all/vars.yml (hunk 1 above)
Comment (adborden) [reply ansible/inventories/bionic/group_vars/all/vars.yml]: Very poor naming, and that is why I had accidentally mixed them up the first time around.

I'll rename these:
- `_public_key`-> `_public_certificate`
- `_private_pem` -> `_private_key`

**Comment by adborden** [comment]
Rebased to resolve merge conflicts. This is ready for another review

**Review by pburkholder — APPROVED** [review]

**Pull request merged**